app.rb

```diff
@@ -40,8 +110,10 @@ def list_all_people
       end
     end
   end
+end
 
-  def create_a_person
+class CreateList
+  def create_a_person(people)
     puts 'Do you want to create a student(1) or a teacher (2)?'
     choice = user_input('[Input the number]:  ')
```

Comment:
- [ ] I think you didn't refactor this since here you shouldn't ask for options or validation you can also refactor it in another file and just call the validtion ethod here? by that you are making you code clean and follow `Single-responsibility Principle` each file responsiblie for one functionality 💪🏻🚀💯🔥 so kindly refactor as much as you can ✅🙏🏻